pkg/plugin/interfaces.go

```diff
@@ -95,3 +95,13 @@ type CreateWebhookPluginGetter interface {
 type CreateWebhook interface {
 	GenericSubcommand
 }
+
+type EditPluginGetter interface {
```

Comment:
```suggestion
// EditPluginGetter is an interface that defines gets an Edit plugin
type EditPluginGetter interface {
```

Reply:
Done

Reply:
Hi @prafull01,

It is missing the comment.